Flask_webapp/templates/index.html

```diff
@@ -101,6 +100,9 @@
 <li>Provide a valid stock ticker.</li>
 <li>Select a model. </li>
 
+<br>
+<p style="color:red; font-size:30px">{{string}}</p>
```

Comment:
I would opt for a more meaningful variable name, `error_message` instead of `string`

Reply:
also, looks like you did the color and font here?